.editorconfig

```diff
@@ -4,10 +4,13 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 charset = utf-8
-indent_size = 2
+indent_size = 1
```

Comment:
Does this change cause any changes in the actual spec? If so, should we be including it as part of this change?

Reply:
That's a good question. Happy to leave that out this round, but I do think it would be good for Console to align on this.

Reply:
Sorry, I meant should we be including the changes _to the spec_ as part of this. I don't use editorconfig, so it will take me a bit to find out myself, and it's clear I should already be sleeping. ;)

Reply:
If someone is willing to reformat the specification we could include that change here I suppose, but it might be better as a distinct commit.